### PR TITLE
[#2260] Removed StatusTranslation model

### PR DESCRIPTION
**Pull request by Bartvaderkin**

### CODE REVIEW

File: src/open_inwoner/openzaak/api_models.py [hunk 1]
```diff
@@ -69,17 +71,14 @@ def process_data(self) -> dict:
...
             "identification": self.identification,
             "uuid": str(self.uuid),
             "start_date": self.startdatum,
             "end_date": getattr(self, "einddatum", None),
             "description": self.zaaktype.omschrijving,
-            "current_status": status_translate.from_glom_multiple(
+            "current_status": glom_multiple(
                 self,
                 ("status.statustype.statustekst", "status.statustype.omschrijving"),
                 default="",
```
Comment (pi-sigma) [inline src/open_inwoner/openzaak/api_models.py:84]: For the `label` in `get_statuses_data` we have `default=_("No data available")`. Should we use this too for the `current_status`, or should the default be an empty string?

File: src/open_inwoner/openzaak/tests/test_notification_utils.py [hunk 2]
```diff
@@ -65,7 +60,7 @@ def test_send_case_update_email(self):
         body_html = email.alternatives[0][0]
         self.assertIn(case.identificatie, body_html)
         self.assertIn(case.zaaktype.omschrijving, body_html)
-        self.assertIn("My Translated Status", body_html)
+        self.assertIn(status.statustype.omschrijving, body_html)
```
Comment (pi-sigma) [inline src/open_inwoner/openzaak/tests/test_notification_utils.py:63]: Are we showing the `omschrijving` or the `statustekst`? They could be the same for some statustypes, different for others.

File: src/open_inwoner/userfeed/tests/hooks/test_case_status.py [hunk 3]
```diff
@@ -99,7 +92,7 @@ def test_status_update(self, mock_get_active_app_names: Mock):
             strip_tags(item.message),
             escape(
                 _("Case status has been changed to '{status}'").format(
-                    status="translated status"
+                    status=status2.statustype.omschrijving
```
Comment (pi-sigma) [inline src/open_inwoner/userfeed/tests/hooks/test_case_status.py:95]: See previous comment about `omschrijving` vs `statustekst`

**Review by pi-sigma — CHANGES_REQUESTED** [review]
Nice cleanup! Couple minor questions.

**Comment by alextreme** [comment]
Discussed with Groningen, and to ease the transition I'm delaying this PR until 1.17

File: src/open_inwoner/openzaak/api_models.py (hunk 1 above)
Comment (Bartvaderkin) [reply src/open_inwoner/openzaak/api_models.py]: Nah, it should be the same so I added it.

File: src/open_inwoner/openzaak/tests/test_notification_utils.py (hunk 2 above)
Comment (Bartvaderkin) [reply src/open_inwoner/openzaak/tests/test_notification_utils.py]: Yes you are right, statustekst has priority, I updated the test

**Review by pi-sigma — APPROVED** [review]

**Pull request merged**